agent/build.gradle

```diff
@@ -0,0 +1,36 @@
+
+plugins {
+    id 'com.github.johnrengelman.shadow' version '1.2.1'
+}
+
+apply from: "$buildScriptsDir/common-java.gradle"
+apply from: "$buildScriptsDir/publishing.gradle"
+
+shadowJar {
+    classifier = ''
+    relocate 'org.objectweb.asm', 'com.microsoft.applicationinsights.agent.dependencies.asm'
+}
+archivesBaseName = 'applicationinsights-agent'
+
+jar {
+    dependsOn shadowJar
+    enabled = false
+    manifest {
+        attributes("Agent-Class": "com.microsoft.applicationinsights.agent.internal.agent.AgentImplementation",
```

Comment:
Wouldn't "Premain-Class" suffice here?
I.e. do we need both "Agent-Class" and "Premain-Class"?